lib/hmac.js

```diff
@@ -11,7 +11,7 @@ const bufferEq = require('buffer-equal-constant-time');
 const createHmac = (key, text) => {
     const hmac = crypto.createHmac('sha1', key);
     hmac.setEncoding('hex');
-    hmac.write(text);
+    hmac.write(new Buffer(text, 'utf-8'));
```

Comment:
Interesting, I wonder if this is the same as `hmac.write(new Buffer(text, 'utf-8'));`